tsconfig.build.json

```diff
@@ -1,11 +1,11 @@
 {
   "references": [
-    { "path": "./packages/design-system-react-native/tsconfig.build.json" },
-    { "path": "./packages/design-system-react/tsconfig.build.json" },
     { "path": "./packages/design-system-shared/tsconfig.build.json" },
+    { "path": "./packages/design-tokens/tsconfig.build.json" },
     { "path": "./packages/design-system-tailwind-preset/tsconfig.build.json" },
     { "path": "./packages/design-system-twrnc-preset/tsconfig.build.json" },
-    { "path": "./packages/design-tokens/tsconfig.build.json" }
+    { "path": "./packages/design-system-react/tsconfig.build.json" },
+    { "path": "./packages/design-system-react-native/tsconfig.build.json" }
```

Comment:
The rearranging of dependencies probably is uncessary but updating typescript references to include `design-system-shared`